### DISCUSSION TITLE
C'est quoi @Output ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#26] [General]
# C'est quoi @Output ? | Bonjour Angular

C'est quoi @Output dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/output/

**Comment by vnivuahc** [comment]
Bonjour, je ne sais pas si derrière cela utilise un Signal, mais le `@Output` peut désormais être remplacé par `output()`.

```typescript
@Output() customEvent = new EventEmitter<string>();
// devient
customEvent= output<Book>();

// et l’utilisation est similaire
this.customEvent.emit('Hello Parent!');
```

Source : [https://blog.angular-university.io/angular-signal-components/](https://blog.angular-university.io/angular-signal-components/#angular-component-outputs-with-output)
